deepspeed/checkpoint/deepspeed_checkpoint.py

```diff
@@ -263,11 +270,13 @@ def validate_files(self):
 
     def _get_layer_keys(self):
         key_set = set()
-        key_len = len(LAYER_FILE_PREFIX) + 2
         for file_path in self.layer_files:
             _, fname = os.path.split(file_path)
-            key_set.add(fname[:key_len])
-        return sorted(list(key_set))
+            layer_id = re.search(r'layer_(\d+)-model_.*', fname).group(1)
```

Comment:
Is it possible to incorporate symbolic constants, such as `LAYER_FILE_PREFIX`, into this search pattern instead of using hardcoded string?

Reply:
@tjruwase Done.